src/Sulu/Bundle/CustomUrlBundle/Controller/CustomUrlController.php

```diff
@@ -95,7 +95,7 @@ public function postAction($webspaceKey, Request $request)
         $document = $this->get('sulu_custom_urls.manager')->create(
             $webspaceKey,
             $request->request->all(),
-            $this->getLocale($request)
+            $this->getRequestParameter($request, 'targetLocale', true)
```

Comment:
I'd say that the locale is mandatory, as both getAction and cgetAction fail with 500 when no locale is passed. I'll add a test for both of them.